lua/entities/gmod_wire_expression2/base/tokenizer.lua

```diff
@@ -261,17 +249,12 @@ function Tokenizer:Next()
 				if m:sub( -1, -1) == "\"" then
 					break
 				else -- Escape
-					local c = self:At()
+					local c = WireLib.ParseEscapes("\\" .. self:ConsumePattern(".[^\\\"]?[^\\\"]?[^\\\"]?[^\\\"]?[^\\\"]?[^\\\"]?[^\\\"]?}?", true))
 
-					if not Escapes[c] then
-						self:Warning("Invalid escape \\" .. c)
-						c = '\\' .. c
-					else
-						c = Escapes[c]
+					if c:sub(1, 1) == "\\" then
```

Comment:
Will cause the \\ escape to be an error. Maybe store the input in a local and check if the input is equal to the output?